### PR TITLE
[UI] Added historical chart for single observations

### PR DESCRIPTION
**Pull request by sahirgomez1**
@ChrisCummins just in case we need this feature.

This PR

- Cleans debugging lines,
- adds license header to index.html
- adds historical chart for single observations in global historical table/chart.

https://user-images.githubusercontent.com/84393628/133364569-b9becf55-00fe-437b-b4be-72ec56d08de2.mov

### CODE REVIEW

File: www/frontends/compiler_gym/src/components/Navbars/ActionsNavbar.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ const ActionsNavbar = ({
           "1"
         );
         setSession(response);
-        console.log(response)
+        //console.log(response) For debugging.
```
Comment (ChrisCummins) [inline www/frontends/compiler_gym/src/components/Navbars/ActionsNavbar.js]: Nitpick: don't commit commented-out code

File: www/frontends/compiler_gym/src/components/StateVisualizations/ModalSingleChart.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
```
Comment (ChrisCummins) [inline www/frontends/compiler_gym/src/components/StateVisualizations/ModalSingleChart.js:7]: This needs a license header

**Review by ChrisCummins — APPROVED** [review]
LGTM (pending a couple of tiny things, see inline comments), thanks @sahirgomez1!

I love the larger trend graph view!

Cheers,
Chris

File: www/frontends/compiler_gym/src/components/StateVisualizations/ModalSingleChart.js (hunk 2 above)
Comment (sahirgomez1) [reply www/frontends/compiler_gym/src/components/StateVisualizations/ModalSingleChart.js:7]: Ops! good catch.

**Comment by ChrisCummins** [comment]
Changes LGTM, thanks @sahirgomez1. Merge when you're ready!

Cheers,
Chris

**Pull request merged**